app/controllers/users_controller.rb

```diff
@@ -61,7 +61,13 @@ def update
     else
       Honeycomb.add_field("error", @user.errors.messages.reject { |_, v| v.empty? })
       Honeycomb.add_field("errored", true)
-      render :edit, status: :bad_request
+
+      if @tab
+        render :edit, status: :bad_request
+      else
+        flash[:error] = @user.errors.full_messages.join(", ")
```

Comment:
You can use `@user.errors_as_sentence` here

Reply:
I knew there was a better shortcut!!!

Reply:
@mstruve Rails offers `to_sentence` as an alternative to `join(", ")`. During a previous cooldown I wrapped it as  `ApplicationRecord#errors_as_sentence` so we standardize on a snigle format. :smiley: